ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/invitationsServiceAgent.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable, Inject } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { NotAuthenticatedServiceAgent } from './notAuthenticatedServiceAgent.service';
+import { ConfigurationService } from '../configuration.service';
+import { LoggingService } from '../logging.service';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Injectable()
+export class InvitationsServiceAgent extends NotAuthenticatedServiceAgent<any> {
+    constructor(
+        @Inject('ddp.config') _configuration: ConfigurationService,
+        _http: HttpClient,
+        _logger: LoggingService) {
+        super(_configuration, _http, _logger);
+    }
+
+    public verify(invitationId: string): Observable<number | null> {
+        return this.postObservable('/invitations/verify', { invitationId })
+            .pipe(map(response => response && response.status ? response.status : null));
+    }
```

Comment:
See my comment above about handing status code. 
And if this is really just a "verify worked"/"verify did not work" type of call, perhaps a Observable<boolean> would be more appropriate?

Reply:
`empty()` appears to be deprecated in favor of `EMPTY`.

Reply:
Good catch, thanks.